interfaces/builtin/libvirt.go

```diff
@@ -35,6 +36,8 @@ sendto
 sendmsg
 socket
 socketpair
+listen
+accept
```

Comment:
I need to dig into that more - without these stanza's the python process goes defunct on opening the connection to libvirt.